web/src/components/Icon/Icon.tsx

```diff
@@ -64,59 +47,43 @@ export const iconMap = {
 type IconType = keyof typeof iconMap;
 type IconSizeType = "xs" | "sm" | "md" | "lg" | number;
 
-export const Icon = ({ icon, color, size, rotate, ...rest }: Props) => {
-  const IconSvg = iconMap[icon];
-  if (!IconSvg) return null;
-  return (
-    <StyledIcon
-      className="icon"
-      iconColor={color}
-      iconSize={size}
-      rotate={rotate}
-      {...rest}
-    >
-      <IconSvg />
-    </StyledIcon>
-  );
-};
-
-type StyledProps = {
-  iconColor?: keyof typeof colors;
-  iconSize?: IconSizeType;
-  rotate?: string;
-};
-
 const iconSizeMap = {
-  xs: "12px",
-  sm: "16px",
-  md: "24px",
-  lg: "40px",
+  xs: "w-3 h-3",
+  sm: "w-4 h-4",
+  md: "w-6 h-6",
+  lg: "w-10 h-10",
 };
-
 const getDimensions = (iconSize: IconSizeType) => {
-  if (typeof iconSize === "number") return `${iconSize}px`;
+  if (typeof iconSize === "number") return `w-[${iconSize}px] h-[${iconSize}px]`;
```

Comment:
I don't believe this will actually work with tailwind because of its compilation process
https://tailwindcss.com/docs/content-configuration#dynamic-class-names